src/components/ConfirmBox/ConfirmBox.tsx

```diff
@@ -0,0 +1,24 @@
+import Button from "../Button/Button";
+import Modal from "../Modal/Modal";
+
+import ConfirmBoxWrapper from "./styles";
+
+const ConfirmBox = () => {
+  return (
+    <Modal>
+      <ConfirmBoxWrapper>
+        <h2>Delete Employee Details</h2>
```

Comment:
Either use styled component or class name. Make it consistant

Reply:
Correction done in Phase-1 branch.